src/ui/hooks/useDetailsTabs.tsx

```diff
@@ -47,6 +47,9 @@ export function useDetailsTabs(currentStatus: Status) {
           return (
             <>
               {!failedReason && <div className="error">{'NA'}</div>}
+              {stacktrace && stacktrace.length === 0 && (
```

Comment:
If the `stacktrace` is empty there is no good reason to render `<Highlight>`...
It should look like
```js
     {stacktrace?.length === 0 ? (
        <div className="error">{!!failedReason ? failedReason : "NA"}</div>
      ) : (
        <Highlight language="stacktrace" key="stacktrace">
          {stacktrace}
        </Highlight>
      )}
```